OKE/kubernetes-comparison.md

```diff
@@ -1,28 +1,29 @@
 ---
-title: "Kubernetes: A comparison of managed engines"
+title: Kubernetes: A comparison of managed engines
 description: An extensive comparison of the four most prolific managed Kubernetes offerings.
 author:
   name: Eli Schilling
   bio: Oracle DevRel Advocate, cloud native app dev and DevOps.
-  github: "https://github.com/oracle-devrel"
-  youtube: "https://www.youtube.com/c/oracledevs/playlists"
+  github: https://github.com/oracle-devrel
```

Comment:
These URLs should probably remain double quoted, as the colon may throw off YAML processing